python/cuspatial/cuspatial/tests/binpreds/test_equals_only_binpreds.py

```diff
@@ -548,21 +482,34 @@ def test_3_linestrings_geom_equals_3_linestrings_longer():
     pd.testing.assert_series_equal(expected, got.to_pandas())
 
 
-def test_3_polygons_geom_equals_3_polygons_different_sizes():
+def test_pair_linestrings_different_last_two():
+    gpdlinestring1 = gpd.GeoSeries(
+        [
+            LineString([(0, 0), (1, 1), (2, 1)]),
+        ]
+    )
+    gpdlinestring2 = gpd.GeoSeries(
+        [
+            LineString([(0, 0), (2, 1), (1, 1)]),
+        ]
+    )
+    linestring1 = cuspatial.from_geopandas(gpdlinestring1)
+    linestring2 = cuspatial.from_geopandas(gpdlinestring2)
+    got = linestring1.geom_equals(linestring2)
+    expected = gpdlinestring1.geom_equals(gpdlinestring2)
+    pd.testing.assert_series_equal(expected, got.to_pandas())
+
+
+"""These tests will be reintroduced when we have `.contains` implemented
+def test_pair_polygons_different_ordering():
```

Comment:
Use `pytest.skip(reason =...)`, not comment out.

Reply:
Done.

Reply:
It's not shown as done here. Have you pushed it?

Reply:
Hmm I thought so.